src/dbnode/persist/fs/msgpack/encoder_test.go

```diff
@@ -40,19 +40,33 @@ func testCapturingEncoder(t *testing.T) (*Encoder, *[]interface{}) {
 	encoder := NewEncoder()
 
 	var result []interface{}
+	actualEncodeVarintFn := encoder.encodeVarintFn
```

Comment:
Capturing encoder previously captured result but didn't write through to buffer. Since the buffer is needed to calculate the checksum, ensure original method is called as well